README.md

```diff
@@ -1,14 +1,30 @@
+
 # CircuitPython organization site
+A more detailed gude for adding a board to CircuitPython can be found in the
+following Adafruit Learn guide: [How to add a New Board to the circuitpython.org website](https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website)
 
 **To add a new board to the site:**
 
 1. Duplicate `template.md` to `_board/<board id>.md`.
 2. Edit `_board/<board id>.md` according to the template's instructions.
-3. Provide 3 images. An original high-quality image. A smaller image (300 px width),
-and a larger image (700 px width) in each respective directory (assets/images/boards/{small large original})
-and process them in something like https://squoosh.app/ to reduce file size. If
+3. In your `_board/<board id>.md` you will specifigy a `board_image`. Create 3
+versions of this file, in the following sizes and folder locations:
+
+|Size|File|Dimensions (px)|
+|--|--|--|
+|Original|assets/images/orignal/{board_image}|900px+ width and 4:3 ratio width|
```

Comment:
Some of the images are 13:10 which is fine. Min widths are 292px for small and 590px for large, but these are absolute minimums and could be a little bigger like you suggested. Original Image minimum size is the same as the large, and 900px or larger is sometimes difficult or impossible to find.

Reply:
I added a note about seeing the docs for 13:10.